### PR TITLE
Feature/fp21 11 landing page background container

### PR DESCRIPTION
**Pull request by rebeccahockley**
Added background container with CSS styles according to Figma stylings. Have added one media query for responsiveness and will add to this as we progress through the project. 

Run the code and look in the browser to see the different views at various windows widths.

### CODE REVIEW

**Review by te-online — COMMENTED** [review]
Good work 👍 I left two comments below. Didn't do any functional testing, maybe you could ask someone from another team to do that? 🙂

File: packages/client/src/components/Main/Main.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+main {
+  background: linear-gradient(
+      180deg,
+      rgba(28, 56, 64, 0.7) 0%,
+      rgba(28, 65, 64, 0.8) 0.01%,
+      rgba(7, 27, 32, 0.5) 100%
+    ),
+    url('../../../public/assets/images/body-background-image-concrete.png');
```
[the remark this answers is not in the document]
Comment (mag-da-lenka) [reply packages/client/src/components/Main/Main.css:8]: Ehh, those heavy concrete structures. Good news though :D 
image

[the remark this answers is not in the document]
Comment (mag-da-lenka) [reply packages/client/src/components/Main/Main.css:8]: image

**Review by te-online — APPROVED** [review]

**Review by ricardoaguiar — APPROVED** [review]

File: packages/client/src/containers/LandingPage/LandingPage.Style.css [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .landing-page-container {
-  color: #687808;
+  color: #ffffff;
```
Comment (oleksandra-lavryk) [inline packages/client/src/containers/LandingPage/LandingPage.Style.css]: @rebeccahockley  Hi. Everything looks good. But we need to change color format to RGB.

**Review by oleksandra-lavryk — CHANGES_REQUESTED** [review]
Take a look at the comment below.

**Review by oleksandra-lavryk — APPROVED** [review]
👍

**Pull request merged**